input/pagecontent/changes.md

```diff
@@ -101,5 +101,7 @@ This change log documents the significant updates and resolutions implemented fr
   - name search parameter to be SHOULD instead of SHALL [FHIR-45133](https://jira.hl7.org/browse/FHIR-45133)
 - Made the following changes to AU Core Diagnostic Result Observation:
   - in Observation.performer replaced RelatedPerson with AU Base RelatedPerson [FHIR-45228](https://jira.hl7.org/browse/FHIR-45228)
+  - removed the value set constraint on Observation.status [FHIR-45125](https://jira.hl7.org/browse/FHIR-45125)
 - Made the following changes to AU Core Pathology Result Observation:
-  - in Observation.performer replaced RelatedPerson with AU Base RelatedPerson [FHIR-45228](https://jira.hl7.org/browse/FHIR-45228)
\ No newline at end of file
+  - in Observation.performer replaced RelatedPerson with AU Base RelatedPerson [FHIR-45228](https://jira.hl7.org/browse/FHIR-45228)
+  - removed the value set constraint on Observation.status [FHIR-45125](https://jira.hl7.org/browse/FHIR-45125)
```

Comment:
Not clear enough. Say which value set was removed.

Reply:
Done.